src/setup_tests.ts

```diff
@@ -4,6 +4,9 @@
  * SPDX-License-Identifier: MIT
  */
 
+import * as matchers from 'vitest-dom/matchers';
+import 'vitest-dom/extend-expect';
+
```

Comment:
I don't think these are used?

Reply:
I think we want the second one as step 4 of https://testing-library.com/docs/svelte-testing-library/setup

I'd accidentally done both of the options for that step and then not quite cleaned up afterwards. We're not actually using them yet but seemed prudent to follow the steps. Tidied in 4472aa4a2838bf7c217992c1bd00e5729f4e6a6e